extensions/wikia/Discussions/maintenance/dumpForumData.php

```diff
@@ -0,0 +1,97 @@
+<?php
+/**
+ * Dumps the Forum data for the site selected by setting the SERVER_ID variable
+ *
+ * @ingroup Maintenance
+ */
+
+error_reporting(E_ALL);
+
+require_once( __DIR__ . '/../../../../maintenance/Maintenance.php' );
+require_once( __DIR__ . '/ForumDumper.php' );
+
+class DumpForumData extends Maintenance {
+	private $dumper;
+
+	public function __construct() {
+		parent::__construct();
+		$this->mDescription = 'Dumps a set of INSERT statements suitable for importing into Discussion "import" tables';
+
+		$this->dumper = new Discussions\ForumDumper();
+	}
+
+	public function execute() {
+
+		$this->clearImportTables();
+		$this->dumpPages();
+		$this->dumpRevisions();
+		$this->dumpVotes();
+	}
+
+	public function clearImportTables() {
+		echo "DELETE FROM import_page;\n";
+		echo "DELETE FROM import_revision;\n";
+		echo "DELETE FROM import_vote;\n";
+	}
+
+	public function dumpPages() {
+		$pages = $this->dumper->getPages();
+		
+		foreach ( $pages as $id => $data ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "$id" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1481)